packages/process/process-monitor.pony

```diff
@@ -100,18 +100,26 @@ use @pony_asio_event_create[AsioEventID](owner: AsioEventNotify, fd: U32,
 use @pony_asio_event_unsubscribe[None](event: AsioEventID)
 use @pony_asio_event_destroy[None](event: AsioEventID)
       
-primitive _EINTR        fun apply(): I32 => 4
-primitive _EAGAIN       fun apply(): I32 => 35
-primitive _STDINFILENO  fun apply(): U32 => 0
-primitive _STDOUTFILENO fun apply(): U32 => 1
-primitive _STDERRFILENO fun apply(): U32 => 2
-primitive _ONONBLOCK    fun apply(): I32 => 4
-primitive _FSETFL       fun apply(): I32 => 4
-primitive _FGETFL       fun apply(): I32 => 3
-primitive _FSETFD       fun apply(): I32 => 2
-primitive _FGETFD       fun apply(): I32 => 1
-primitive _FDCLOEXEC    fun apply(): I32 => 1
-primitive _SIGTERM      fun apply(): I32 => 15
+primitive _EINTR          fun apply(): I32 => 4
+primitive _STDINFILENO    fun apply(): U32 => 0
+primitive _STDOUTFILENO   fun apply(): U32 => 1
+primitive _STDERRFILENO   fun apply(): U32 => 2
+primitive _FSETFL         fun apply(): I32 => 4
+primitive _FGETFL         fun apply(): I32 => 3
+primitive _FSETFD         fun apply(): I32 => 2
+primitive _FGETFD         fun apply(): I32 => 1
+primitive _FDCLOEXEC      fun apply(): I32 => 1
+primitive _SIGTERM        fun apply(): I32 => 15
+
+primitive _EAGAIN      fun apply(): I32 =>
```

Comment:
Since these primitives are longer than one line, I think they should use the standard Pony formatting conventions, since there's no hope for aligning the numbers at the end.